cmd/tapcli/universe.go

```diff
@@ -74,7 +73,11 @@ var universeRootsCommand = cli.Command{
 	Action: universeRoots,
 }
 
-func parseUniverseID(ctx *cli.Context, mustParse bool) (*universerpc.ID, error) {
+func parseProofType(ctx *cli.Context) (*unirpc.ProofType, error) {
+	if !ctx.IsSet(proofTypeName) {
```

Comment:
IIUC we set a default for issuance above, it always _should_ be set?

Reply:
Ah that came in after a rebase, will remove.

Reply:
Moving to no default for these commands to match the other flags.